manifests/params.pp

```diff
@@ -68,7 +68,7 @@
   }
 
   # Zabbix overall params. Is used by all components.
-  $zabbix_package_state                     = 'present'
+  $zabbix_package_state                     = 'latest'
```

Comment:
yeah, but this would then just happen whilst zabbix_version would still say 2.4 or whatever.  Incompatible configuration would then be applied.  I think you need a separate parameter for package_version.